src/main/java/hardcorequesting/team/PlayerEntry.java

```diff
@@ -118,7 +119,6 @@ public void write(JsonWriter out) throws IOException {
         out.endObject();
     }
 
-    @SideOnly(Side.SERVER)
```

Comment:
100% agree but the client continued to crash with this line in place, stating the method was missing. Mainly when trying to pick up entity items.

Also, I've found json recipes tend to be temperamental and not always work, so used the built in GameRegistry recipe registration.

Feel free to reject the Russina/Chinese translation files - just need to update the names of the items (couldn't get rendering working when they were metadata items)